README.md

```diff
@@ -1,18 +1,40 @@
 # Kotlin Android Architecture Library - Kaal - by eMan
 
+[![Latest version](https://img.shields.io/github/v/release/eManPrague/kaal)](https://github.com/eManPrague/kaal/releases/tag/v0.8.0)
```

Comment:
Není možné mít odkaz co by šel rovnou na nejnovější verzi? Je naprd vždy když knihovna má v readme nějakou verzi ale existuje už novější.

Reply:
Ten badge by měl vždy ukazovat na tu nejnovější verzi. Jestli má GitHub API, aby šel udělat i ten odkaz dynamicky, tak to nevím, ale můžu se zkusit podívat.